### PR TITLE
added all the dropdown menus and connected them to the backend

### PR DESCRIPTION
**Pull request by yaay**

### CODE REVIEW

**Review by Rawan2211 — APPROVED** [review]

File: src/app/domain/azhar/azhar.repository.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    constructor(http: HttpClient) {
+        super(http)
+    }
+
+    getResourceUrl(): string {
+        return 'governorate'
+    }
+
+
+    getAzhar(governorateId: any): Observable<any> {
```
Comment (Tasneem-Gad) [inline src/app/domain/azhar/azhar.repository.ts:20]: don't use any.

File: src/app/domain/club/club.repository.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export class ClubRepository extends ResourceService {
+    constructor(http: HttpClient) {
+        super(http)
+    }
+
+    getResourceUrl(): string {
+        return 'governorate'
+    }
+
+    getClubs(governorateId: any): Observable<any> {
```
Comment (Tasneem-Gad) [inline src/app/domain/club/club.repository.ts:19]: same attract type

File: src/app/domain/club/models/club.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export interface Club {
+    "id": number,
+    "version": number,
+    "arabicName": string,
+    "englishName": string,
+    "code": string,
+    "enabled": boolean,
+    "governorate": {
```
Comment (Tasneem-Gad) [inline src/app/domain/club/models/club.ts:8]: same

File: src/app/domain/eduction-administration/model/eduction-administration.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export interface EducationAdministration {
+    "id": number,
+    "version": number,
+    "arabicName": string,
+    "englishName": string,
+    "code": string,
+    "enabled": boolean,
+    "governorate": {
```
Comment (Tasneem-Gad) [inline src/app/domain/eduction-administration/model/eduction-administration.ts:8]: same

**Review by Tasneem-Gad — CHANGES_REQUESTED** [review]

**Comment by yaay** [comment]
Fixed all issues.

**Pull request merged**